core/controllers/editor.py

```diff
@@ -324,11 +323,9 @@ def _get_exploration_data(
 
         return editor_dict
 
+    @base.require_user
```

Comment:
Ok. So should I shift to the behaviour - "anyone who can play an exploration can view it's editor page"?